docs/index.yaml

```diff
@@ -327,7 +355,8 @@ entries:
   - apiVersion: v1
     appVersion: "3.0"
     created: "2020-05-13T15:47:43.269014-04:00"
-    description: A Helm chart for installing the HPE Volume Driver for Kubernetes FlexVolume plugin
+    description: A Helm chart for installing the HPE Volume Driver for Kubernetes
+      FlexVolume plugin
```

Comment:
yup I haven't touched the flex volume chart. the line break causes the timestamp change. even if I revert it now, next time it would be autogenerated. for this PR I can manually revert it.